bgp/runner.py

```diff
@@ -49,8 +49,11 @@ def sequence(self, book):
         'term_freq': WordFreqModule(),
         'isbns': IsbnExtractorModule(),
         'urls': UrlExtractorModule()
-    }, n=1, stop_words=None)
-})
+    }, n=1, stop_words=None),
```

Comment:
1. At this point, we may want to add a small test for the book `TheGeometry` and ensure the results do contain `DESCARTES`. 
2. Similar to @tabshaikh's question above, the KeywordPageDetectorModule should take keyword as an argument, not the `PageTypeProcessor`
Also, I imagine we probably want to `.lower()` our results when we search to increase the probability of matches (or at least provide this as an option).

Reply:
2nd - done
Keyword is passed from` KeywordPageDetectorModule` now..
Also removed case sensitivity using `.lower`

Reply:
lgtm!